apps/worker/src/app/workflow/services/workflow-queue.service.ts

```diff
@@ -99,30 +99,44 @@ export class WorkflowQueueService extends QueueService<IJobData> implements INov
     };
   }
 
-  private async jobHasCompleted(job): Promise<void> {
+  private async jobHasCompleted(job: Job<IJobData, any, string>): Promise<void> {
+    let jobId;
+
     try {
+      jobId = job.data._id;
+      const environmentId = job.data._environmentId;
+      const userId = job.data._userId;
+
       await this.setJobAsCompleted.execute(
         SetJobAsCommand.create({
-          environmentId: job.data._environmentId,
-          _jobId: job.data._id,
-          organizationId: job.data._organizationId,
+          environmentId,
+          jobId,
+          userId,
         })
       );
     } catch (error) {
-      Logger.error('Failed to set job as completed', LOG_CONTEXT, error);
+      Logger.error(`Failed to set job ${jobId} as completed`, error, LOG_CONTEXT);
```

Comment:
I realised this is the right order when adding context to error loggings. Need to do a task to fix it all across the codebase.